src/scss/grommet-core/_objects.box.scss

```diff
@@ -9,6 +9,12 @@
       @include media-query (palm) {
         flex-basis: auto;
         max-width: none;
+
+        // IE11 specific fix to prevent width overflow
+        // in proportional boxes in mobile
+        @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+          width: 100%;
```

Comment:
Yes, I've updated _objects.box.scss to apply the proportional box styles in all cases.
https://github.com/grommet/grommet/pull/937/files#diff-33614de6ad927708ac4b9dd6e8d19a0dR12